### PR TITLE
Fix --no-color in ShowPatchAction

### PR DESCRIPTION
**Pull request by Nosferatul**
Closes https://github.com/coala/coala/issues/4618

### CODE REVIEW

**Comment by Nosferatul** [comment]
http://imgur.com/BWzn0me on the left, colors with the change from this pr, on the right the colors with master branch.

File: coalib/results/result_actions/ShowPatchAction.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def apply(self,
               result,
               original_file_dict,
               file_diff_dict,
-              colored: bool=True,
+              no_color: bool=True,
               show_result_on_top: bool=False):
         """
```
Comment (satwikkansal) [inline coalib/results/result_actions/ShowPatchAction.py]: param docs not updated :)

Comment (Nosferatul) [inline coalib/results/result_actions/ShowPatchAction.py]: done :+1:

**Comment by Adrianzatreanu** [comment]
ack f43bd5e

**Comment by Nosferatul** [comment]
@rultor merge

**Pull request merged**